jobs/pipelines/powervs/custom-powervs-jobs/custom-ocp4x-powervs-sydney04-p9-min-next-upgrade/Jenkinsfile

```diff
@@ -0,0 +1,256 @@
+@Library('jenkins-upstream-library') _
+
+pipeline {
+    agent {
+        docker {
+            image 'quay.io/powercloud/inbound-agent:4.6-1.3'
+            args '-v /etc/resolv.conf:/etc/resolv.conf'
+            label 'jump-vpc-x86_64'
+        }
+    }
+    
+    parameters {
+        string(defaultValue: '', description: 'Current Build(quay image or build number)', name: 'CurrentBuild')
+        string(defaultValue: '', description: 'Upgrade Build(quay image or build number)', name: 'UpgradeBuild')
+        string(defaultValue: '', description: 'Current Build Release', name: 'Release')
```

Comment:
@varad-ahirwadkar Please add validation for release and upgrade release

Reply:
ok sure